src/name_service/delivery/utils.clj

```diff
@@ -0,0 +1,46 @@
+(ns name-service.delivery.utils
+  (:require [name-service.core.action :as action]
+            [cheshire.core :as json]))
+
+(defn bad-request
+  [error]
+  {:status 400 :body error})
```

Comment:
Hm. How about changing a little here. For example you could make a module called `http` and in there you could have something like this:

```clj
(defn- response [status body & {:as headers}]
  {:status status :body body :headers (merge {"Content-Type" "application/json"} headers)})

(def ok          (partial response 200))
(def bad-request (partial response 400))
```

and then import the responses and use it like that for example:
```clj
(http/ok <response-body>)
```